Library/Formula/cryptol.rb

```diff
@@ -0,0 +1,42 @@
+require "language/haskell"
+
+class Cryptol < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.cryptol.net/"
+  url "https://github.com/GaloisInc/cryptol/archive/v2.2.0.tar.gz"
+  sha256 "90d2cbe35db8b2a9fcd78eaa2c08ab0cd81641a30949ab855dde71d17429d3ee"
+  head "https://github.com/GaloisInc/cryptol.git"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "cvc4"
+
+  def install
+    cabal_sandbox do
+      if build.stable?
+        # When we're building from the stable tarball, we have to
+        # provide the hash ourselves. Update this when updating the
+        # stable release.
+        system "make", "REV=0536d0f", "PREFIX=#{prefix}", "install"
```

Comment:
When I've tried that, I run into the fact that the working copy gets checked out into another directory:

```
==> Checking out tag v2.2.0
git checkout-index -a -f --prefix=/private/tmp/cryptol-Gt0CsI/
```

Since the `.git` directory isn't local, when the Cryptol build process tries to run `git rev-parse` it fails because it's not in a repo. I tried poking around the various methods on the `stable` object but couldn't find an obvious way to find that directory or query the current checkout for status